tests/library/chromium/launcher.spec.ts

```diff
@@ -184,13 +184,3 @@ it('should not create pages automatically', async ({ browserType }) => {
   await browser.close();
   expect(targets.length).toBe(0);
 });
-
-it('should fallback to regular chromium when running chromium-headless-shell channel as headed', async ({ browserType, channel }) => {
```

Comment:
why isn't this test needed anymore?

Reply:
The behavior it tests is no longer true.